### PR TITLE
Update vanilla-0.16.51.json

### PR DESCRIPTION
**Pull request by antomarsi**
Found a string error

### CODE REVIEW

File: data/vanilla-0.16.51.json [hunk 1]
```diff
@@ -9385,4 +9385,4 @@
             "speed": 0.03125
         }
     }
-}
\ No newline at end of file
+}
```
Comment (SuperSandro2000) [inline data/vanilla-0.16.51.json]: I like this change the most. :)

Comment (antomarsi) [reply data/vanilla-0.16.51.json]: yeah, that was a auto-format, sorry.

Comment (SuperSandro2000) [reply data/vanilla-0.16.51.json]: That was no irony. It messes up diffs at the end of the file.

**Comment by RustyBlade64** [comment]
These files are generated automatically from the game. The "tertiary" vs. "terciary" is a naming inconsistency in the modding API that was fixed by Factorio devs in 0.17.0.

Therefore I don't think it's a good idea to change this file. IMHO issues like these should be fixed at the source (i.e. Factorio prototype definitions, which has already happened) and not in auto-generated files derived from them.

**Comment by antomarsi** [comment]
Closing because like @RustyBlade64 said, it was a naming inconsistency.

**Pull request closed without merging**